Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -897,6 +897,7 @@ private extension BrowserViewController {
 
     // Use for sms and mailto links, which do not show a confirmation before opening.
     func showSnackbar(forExternalUrl url: URL, tab: Tab, completion: @escaping (Bool) -> Void) {
+        /* Ecosia: Change to native alert controller
         let snackBar = TimerSnackBar(text: .ExternalLinkGenericConfirmation + "\n\(url.absoluteString)", img: nil)
```

Comment:
Just adding a thought: I think it makes sense to check (and replace) other usages of `TimerSnackBar` and `SnackButton` as they will likely have the same unwanted UX

Reply:
Thanks @ecotopian . On second thought I also replaced the behavior for the external app links so that they will behave the same. For the login credentials prompt, I left it as it is (we do the same on current Ecosia production)